File: Java/program5.java
// Calculate perecntage 

import java.util.*;

class Percentage
{
        float CalculatePercentage (int TotalMarks, int ObtainedMarks)
        {
            float perecntage = 0.0f;
            perecntage = ((float)ObtainedMarks / (float)TotalMarks) * 100;
            return perecntage;
       
        }
}
        

class program5
{
    public static void main(String A[])
    {
        int Total = 0, Obtained = 0;
        float iRet = 0.0f;
       

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter your total marks :");
        Total = sobj.nextInt();

        System.out.println("Enter your obtained marks :");
        Obtained = sobj.nextInt();

        Percentage pobj = new Percentage();
        iRet = pobj.CalculatePercentage(Total,Obtained);

         System.out.println("Percentage is :"+iRet);

  
        
    }
}

File: C/program5.c
///////////////////////////////////////////////////////////////////////////////
// File Name   :  program2.c
// Description :  Used to calculate percentage
// Autor       :  Ankita Anil Patil
// Date        :  28/04/2025
///////////////////////////////////////////////////////////////////////////////


#include<stdio.h>

float CalculatePercentage(int iTotalMarks, int iObtainedMarks)
{
    float fPercentageValue = 0.0f;
    fPercentageValue=((float)iObtainedMarks / (float) iTotalMarks) * 100 ;
    return fPercentageValue;
}

int main()
{
    int iTotal =0;
    int iObtained =0;
    float fPercentage=0.0f;

    printf("Enter your Total marks : ");
    scanf("%d",&iTotal);
    
    printf("Enter your obtained marks :");
    scanf("%d",&iObtained);

    //
    // percentage = ( (float) obtained / (float) Total ) * 100;
    fPercentage  = CalculatePercentage (iTotal ,iObtained);
    printf("Percentage is : %.2f%%\n",fPercentage);
    return 0;
}